### ISSUE TITLE
wcs:GetCapabilities_AcceptVersions_0.0.0 test is faulty

### ISSUE THREAD

**Issue by pomakis** [opengeospatial/ets-wcs11#20]
The wcs:GetCapabilities_AcceptVersions_0.0.0 test verifies "that when a GetCapabilities request with an AcceptVersions parameter with the version number 'too low' is submitted, the response is valid XML that validates against the OWS Common 1.1.0 ExceptionReport schema".  However, in a GetCapabilities request, the AcceptVersions parameter is the only indication a server has as to what WCS versions the client supports.  So if the request doesn't list any versions that the server has support for, the server has no choice but to reply with an exception-report response that's compliant to a WCS version that the client probably won't understand.  It can't magically guess that the client actually does support WCS 1.1.1 but neglected to include that version in the AcceptVersions parameter.  The OWS 1.1.0 specification (OGC 06-121r3), section 7.3.2, anticipates this by stating:

```
To ensure backward compatibility, clients shall also be prepared to 
accept an unknown response and treat this situation as an indication
that version negotiation has failed.
```

So about the only thing a test like this can rightfully do, if anything, is to make sure that what comes back is _not_ a WCS 1.1.1 capabilities document.

(As a side note, I notice that the test also includes a version=1.1.1 parameter.  However, since version is not a valid parameter for GetCapabilities requests, the server shouldn't be expected to pick up on this as a hint for what exception-report version to return.)

**Comment by pomakis** [comment]
The same is true for the wcs:GetCapabilities_AcceptVersions_99.99.99 test.

**Comment by dstenger** [comment]
Thank you for reporting the issue. We will do further investigation.

**Comment by dstenger** [comment]
I agree with your analysis. If the version parameter is missing the server cannot know the version which was meant to be requested (thus, the server cannot know what exception report is expected). This is confirmed, as you already stated, by OWS 1.1.0 specification (OGC 06-121r3), section 7.3.2.

On the other hand, it can be argued that a service which is meant to pass the test suite should just support the version of the ETS and thus return a correct exception report.

We will continue with the discussion and give you feedback soon.

**Comment by dstenger** [comment]
Issue is fixed by pull request #25.
The fix will be part of the next release of the ets.

**Comment by dstenger** [comment]
The fix was successfully tested, merged into master branch and will be part of the next release.
